app/policies/user_policy.rb

```diff
@@ -11,8 +11,12 @@ def create?
     is_admin? || is_himself?
```

Comment:
Yeah, cool! 
I just thought it was funny that at some point it existed a policy for `update?`, then Akita just changed to `create?` resulting in this PR's bug.